ninogomez/taskWorldExam
# Added comments and updated readme

Steps for this test: 
√ Create Project
 √ Create two Tasklists 
 √ Create tasks under newly create Tasklists
 √ Add a comment
 √ Upload a file
 √ Add due date to task
 √ Task overview should display due date, comment and file count
 √ should Copy Project
 √ Copied Project contents should be correct
